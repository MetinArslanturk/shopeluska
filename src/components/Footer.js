import React from 'react';

const Footer = (props) => {
    return (
        <>
            <div>Copyright to <a rel="noopener noreferrer" target="_blank" href="http://www.metinarslanturk.com">Metin Arslanturk</a> You can check <a rel="noopener noreferrer" target="_blank" href="https://github.com/MetinArslanturk/shopeluska">GitHub of This Project</a></div>
        </>
    );
}



export default Footer;