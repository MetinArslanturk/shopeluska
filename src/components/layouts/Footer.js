import React from 'react';

const Footer = () => {
    return (
        <>
            <div>
                Â©{' '}
                <a
                    rel="noopener noreferrer"
                    target="_blank"
                    href="https://www.metinarslanturk.com"
                >
                    Metin Arslanturk
                </a>{' '}
                <span className="mail-text">
                    ( contact@metinarslanturk.com )
                </span>{' '}
                --{' '}
                <a
                    rel="noopener noreferrer"
                    target="_blank"
                    href="https://github.com/MetinArslanturk/shopeluska"
                >
                    GitHub of This Project
                </a>
            </div>
        </>
    );
};

export default Footer;
