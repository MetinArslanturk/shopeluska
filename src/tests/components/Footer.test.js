import React from 'react';
import Footer from '../../components/Footer';
import { shallow } from 'enzyme';

test('should render Footer', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot();
});
