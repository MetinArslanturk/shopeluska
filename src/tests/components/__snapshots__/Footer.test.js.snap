// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer 1`] = `
<Fragment>
  <div>
    Copyright to 
    <a
      href="http://www.metinarslanturk.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      Metin Arslanturk
    </a>
     You can check 
    <a
      href="https://github.com/MetinArslanturk/shopeluska"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub of This Project
    </a>
  </div>
</Fragment>
`;
